File: szxy-funplay-parent/funplay-sellergoods-interface/src/main/java/com/funplay/sellergoods/service/BrandService.java
package com.funplay.sellergoods.service;

import com.funplay.pojo.Brand;
import entity.PageResult;

import java.util.List;
import java.util.Map;


public interface BrandService {

    public List<Brand> findAll();

    public PageResult findPage(Integer page,Integer pageSize);

    public void add(Brand brand);

    public void update(Brand brand);

    public Brand findOne(Long id);

    public void delete(Long [] ids);

    public PageResult findPage(Brand brand, int page,int pageSize);

    List<Map> selectOptionList();
}

File: szxy-funplay-parent/funplay-shop-web/src/main/webapp/js/service/brandService.js
 //品牌服务
    app.service("brandService",function($http){
    	//findAll
    	this.findAll=function(){
    		return $http.get('../brand/findAll.do');
    	}
    	
    	//findPage
    	this.findPage=function(page,size){
    		return $http.get('../brand/findPage.do?page='+page +'&size='+size);
    	}
    	
    	//add
    	this.add=function(entity){
    		return $http.post('../brand/add.do',entity);
    	}
    	
    	//update
    	this.update=function(entity){
    		return $http.post('../brand/update.do',entity);
    	}
    	
    	//findOne
    	this.findOne=function(id){
    		return $http.get("../brand/findOne.do?id="+id);
    	}
    	
    	//del
    	this.del=function(ids){
    		return $http.get("../brand/delete.do?ids="+ids);
    	}
    	//search
    	this.search=function(page,size,searchEntity){
    		return $http.post('../brand/search.do?page='+page+'&rows='+size,searchEntity);
    	}

    	//selectOptionList
		this.selectOptionList=function () {
			return $http.get('../brand/selectOptionList.do');
        }
    })